engine/resources/pippet_run.go

```diff
@@ -0,0 +1,160 @@
+// Mgmt
```

Comment:
I'm not sure that this needs to be in a separate file. Unless you have a good reason, put the contents at the bottom of the pippet.go resource file.

Reply:
Done.